h2olog

```diff
@@ -788,7 +825,14 @@ def handle_quic_event(cpu, data, size):
         build_quic_trace_result(res, ev, ["packet_num"])
     elif ev.type == "h2o__send_response_header":
         build_quic_trace_result(res, ev, ["h2o_conn_id", "h2o_req_id", "h2o_header_name", "h2o_header_value"])
-
+        del res["master_conn_id"]
```

Comment:
I know this is too bad, but we could remove it anyway when we migrate the impl to auto-generated code.

Reply:
This is fine.